.github/workflows/ci_check_translations.yml

```diff
@@ -0,0 +1,43 @@
+name: check translations
+
+on:
+  push:
+    branches:
+      - master
+      - development
+      - release
+  pull_request:
+
+jobs:
+  check_translations:
+    runs-on: ubuntu-18.04
+    services:
+      elasticsearch:
+        image: docker.elastic.co/elasticsearch/elasticsearch-oss:7.10.0-amd64
+        ports:
+          - 9200/tcp
+        options: -e="discovery.type=single-node" --health-cmd="curl http://localhost:9200/_cluster/health" --health-interval=10s --health-timeout=5s --health-retries=10
+      mongodb:
+        image: mongo:4.2.6
+        ports:
+          - 27017/tcp
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js 14.6.x
+        uses: actions/setup-node@v1
+        with:
+          node-version: '14.6.x'
+      - name: Cache node modules
+        uses: actions/cache@v2
+        with:
+          path: ./node_modules
+          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+      - name: install dependencies
+        run: yarn install
+      - run: yarn blank-state
```

Comment:
Do we need the steps after this? Also, can't we get away with just installing the dependencies strictly for the script (babel, and anything else)?

Reply:
Yes, we need the blank-state to check what is in the system context, we could not install other dependencies, but since it's cached from other CI workflows we are not increasing the time